### PR TITLE
Conditional rockets' elements

### PR DESCRIPTION
**Pull request by Alejandro-Bernal-M**
# Rendering the conditional elements on the rockets page and profile component

## Rendering the components conditionally 

image

## Rendering the rockets on my profile
image

### CODE REVIEW

File: src/routes/Profile.jsx [hunk 1]
```diff
@@ -11,21 +11,25 @@ const Profile = () => {
...
+    });
+    return reserved;
+  });
   return (
     <div className="profile-container">
       <ProfileList
         title="Missions"
         elements={reservedMissions}
-        propertyName="mission_name"
+        propertyName="name"
```
Comment (Alejandro-Bernal-M) [inline src/routes/Profile.jsx:28]: The property was changed in order to run the "key" property of the profile list for both arrays

File: src/components/Layout.jsx
Comment (DavidEspinoG) [inline src/components/Layout.jsx:1]: I think it's a great idea to fetch the data on the Layout component. This way, the content is not going to be downloaded every time the user clicks on a link 👏🏼

File: src/components/Rocket.jsx
Comment (DavidEspinoG) [inline src/components/Rocket.jsx:1]: Excellent implementation of the conditional rendering

**Review by DavidEspinoG — APPROVED** [review]
# Hi @Alejandro-Bernal-M  🤝  great job!
 ### Status: 🟢 🟢 🟢 ¡Approved! 🟢 🟢 🟢 
**Please check the comments below for optional recommendations**


## Optional changes: 

It would be nice to add a little space between the reserved button and the description of each rocket 
**Please check the comments below for further information**

**Comment by Alejandro-Bernal-M** [comment]
I added the extra space as you recommended 😄

**Pull request merged**